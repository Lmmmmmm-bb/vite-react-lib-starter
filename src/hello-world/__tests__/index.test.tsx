import HelloWorld from '..';

describe('hello-world', () => {
  it('test', () => {
    const wrapper = <HelloWorld />;
    expect(true).toBeTruthy();
    expect(wrapper).toMatchSnapshot();
  });
});
