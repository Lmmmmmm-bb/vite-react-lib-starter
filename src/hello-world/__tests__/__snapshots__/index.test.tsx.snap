// Vitest Snapshot v1

exports[`hello-world > test 1`] = `<HelloWorld />`;
